---
title: Modelling the zebrafish gut microbiome's resistance and sensitivity to climate
  change and infection
author:
  - Michael J. Sieler Jr.
  - Colleen E. Al-Samarrie
  - Kristin D. Kasschau
  - Mike L. Kent
  - Thomas J. Sharpton
date: "`r format(Sys.time(), '%d %B, %Y')`"
output:
  html_document:
    toc: true
    toc_depth: 3
    toc_float: true
  pdf_document:
    toc: true
    toc_depth: '3'
subtitle: Main Figures and Supplemental Results
---

```{r setup, include=FALSE}
# Setup

knitr::opts_chunk$set(echo = FALSE)
proj.path <- getwd()
path.results <- paste0(proj.path,"/Results") 

```


```{r load-libraries, include=FALSE}
# Load any necessary libraries

library(tidyverse)
library(gt)

```


```{r load-data, include=FALSE}
# Load statistical tables and plots

## Load Stats Objects ------------------------------------------------------

alpha.stats <- readRDS(file = file.path(path.results, "Stats_Plots", "alpha_stats.rds"))
beta.stats <- readRDS(file = file.path(path.results, "Stats_Plots", "beta_stats.rds"))
worm.stats <- readRDS(file = file.path(path.results, "Stats_Plots", "worm_stats.rds"))
diffAbnd.stats <- readRDS(file = file.path(path.results, "Stats_Plots", "diffAbnd_stats.rds"))
randomForest.stats <- readRDS(file = file.path(path.results, "Stats_Plots", "randomForest_stats.rds"))

## Load Plot Objects -------------------------------------------------------

alpha.plots <- readRDS(file = file.path(path.results, "Stats_Plots", "alpha_plots.rds"))
beta.plots <- readRDS(file = file.path(path.results, "Stats_Plots", "beta_plots.rds"))
worm.plots <- readRDS(file = file.path(path.results, "Stats_Plots", "worm_plots.rds"))
diffAbnd.plots <- readRDS(file = file.path(path.results, "Stats_Plots", "diffAbnd_plots.rds"))
randomForest.plots <- readRDS(file = file.path(path.results, "Stats_Plots", "randomForest_plots.rds"))

## Load Session Info -------------------------------------------------------

session_info <- readRDS(file = file.path(path.results, "Stats_Plots", "sessionInfo.rds"))

```


```{r functions, include=FALSE}

# Loads functions required to knit document

source(paste0(proj.path,"/Code/Functions/StartFunctions/plot_settings.R"))
source(paste0(proj.path,"/Code/Functions/HelperFunctions/miscFunctions.R"))

```

```{r plot-saving}

# Create main directory if it doesn't exist
fig_dir <- file.path(proj.path, "Manuscript/Main_Figures")
if (!dir.exists(fig_dir)) dir.create(fig_dir, recursive = TRUE)

# Create separate directories for PNG and PDF files
png_dir <- file.path(fig_dir, "PNG")
pdf_dir <- file.path(fig_dir, "PDF")
if (!dir.exists(png_dir)) dir.create(png_dir, recursive = TRUE)
if (!dir.exists(pdf_dir)) dir.create(pdf_dir, recursive = TRUE)

```

## Overview

### Publication Information

The published version of this manuscript can be found here:
Sieler Jr M.J., Al-Samarrie C.E., Kasschau K.D., Kent M.L., Sharpton T.J.. Modeling the zebrafish gut microbiome's resistance and sensitivity to climate change and infection. *bioRxiv* (2025). DOI: [https://doi.org/10.1101/2025.03.28.644597](https://doi.org/10.1101/2025.03.28.644597)

All code generated during this analysis is available in the [GitHub Repository](https://github.com/sielerjm/Sieler2025__ZF_Temperature_Parasite/).

The raw sequence files generated during the current study are available at the NCBI Sequence Read Archive (SRA) project numbers: [PRJNA1219243](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1219243). 


### Fig. 1) Experimental Design Schematic

```{r exp-scheme, fig.width=7.5, echo=FALSE}

knitr::include_graphics(file.path(proj.path,"Manuscript/Main_Figures/PNG/Fig_1.png"))

```

**Fig.1** Experimental design showing treatments and husbandry events during the course of the study. Symbols indicate when an experimental event occurred at each time point. (1) 260 zebrafish were assigned and acclimated to one of three water temperature groups (e.g., 28°C, 32°C, or 35°C) and reared from 0 to 164 days post fertilization (dpf). (2a) At 164 dpf (or 0 days post exposure; dpe), fecal collections were collected from a random selection of five fish per tank (n = 60). Additionally, histological and wet mount assessments were conducted on selected fish to assess presence of infection and infection burden. (2b) Afterwards, a cohort of fish from each water temperature group were exposed to the nematode Pseudocapillaria tomentosa. (4-6)  Subsequent fecal samples were collected and histopathological assessments were conducted at 14 dpe (n = 54), (4) 21 dpe (n = 48), (5) 28 dpe (n = 47), and (6) 42 dpe (n = 51). 


## Results


### Fig. 2) Water temperature shapes gut microbiome structure {.tabset}


**Fig. 2** Effects of water temperature on zebrafish gut microbiomes. (A) Simpson’s Index of diversity shows that gut microbiome diversity significantly differs between fish reared at 28°C and 35°C water temperatures. (B) Capscale ordination based on the Bray-Curtis dissimilarity of gut microbiome composition constrained on the main effect of temperature. The analysis shows that gut microbiome composition significantly differs between fish reared at different water temperatures. (C) Simpson’s Index of diversity shows microbial gut diversity increases with time from 0 days post exposure (dpe) to 42 dpe, irrespective of water temperature. (D) Capscale ordination of gut microbiome composition based on the Bray-Curtis dissimilarity constrained on the main effects of water temperature and time (days post exposure, dpe), and their interaction. The analysis shows that shows that gut microbiome composition differs between fish across time depending on water temperature. Ribbons and ellipses indicate 95% confidence interval. Only statistically significant relationships are shown. A “*” indicates statistical significance below the “0.05” level. Black arrows indicate direction of greatest change in the indicated by covariates.

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.subset <- "Unexposed"

```

#### 2A)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.variables <- "TEMP"
```


```{r plots-2A, echo=FALSE, fig.height=6.5, fig.width=6.5, message=FALSE, warning=FALSE}

# Store the plot in a temporary variable for saving
p <- alpha.plots[[tmp.subset]][[tmp.variables]][["TUKEY"]][["Plot"]] +
  theme(legend.position = "none") +
  labs(title = NULL)

# Display the plot
p

# Extract figure number from chunk name
fig_name <- "Fig_2A"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-2A}

```


###### GLM

```{r stats-2A-glm, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["GLM.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### ANOVA

```{r stats-2A-anova, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Tukey

```{r stats-2A-tukey, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics



#### 2B)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.variables <- "TEMP"
```


```{r plots-2B, echo=FALSE, fig.height=6.5, fig.width=6.5, message=FALSE, warning=FALSE}

# Store the plot in a temporary variable for saving
p <- beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot"]]

# Display the plot
p

# Extract figure number from chunk name
fig_name <- "Fig_2B"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-2B}

```


###### ADONIS2

```{r stats-2B-adonis2, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (ANOVA)

```{r stats-2B-anova, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (Tukey)

```{r stats-2B-tukey, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics

#### 2C)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.variables <- "TEMP:DPE"
```



```{r plots-2C, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- alpha.plots[[tmp.subset]][[tmp.variables]][["TUKEY"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_2C"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-2C}

```


###### GLM

```{r stats-2C-glm, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["GLM.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### ANOVA

```{r stats-2C-anova, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Tukey

```{r stats-2C-tukey, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics

#### 2D)


```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.variables <- "TEMP:DPE"
```


```{r plots-2D, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_2D"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-2D}

```


###### ADONIS2

```{r stats-2D-adonis2, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (ANOVA)

```{r stats-2D-anova, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (Tukey)

```{r stats-2D-tukey, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics


### {.tabset}

#### (hide)

Click on tabs to reveal figures displaying additional metrics.


#### S2A


```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Unexposed"
tmp.variables <- "TEMP"

```

```{r plots-S2A, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}
alpha.plots[[tmp.subset]][[tmp.variables]][["TUKEY"]][["Plot_SUPP"]]
```

#### S2B

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Unexposed"
tmp.variables <- "TEMP"

```

```{r plots-S2B, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP"]]


```


#### S2C


```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Unexposed"
tmp.variables <- "TEMP:DPE"

```

```{r plots-S2C, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}
alpha.plots[[tmp.subset]][[tmp.variables]][["TUKEY"]][["Plot_SUPP"]]
```

#### S2D

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Unexposed"
tmp.variables <- "TEMP:DPE"

```

```{r plots-S2D, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP"]]

```

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Unexposed"
tmp.variables <- "TEMP:DPE__TIME"

```

```{r plots-S2D.2, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP"]]

```

### Fig. 3) Infection burden is highest in fish reared at lower water temperatures  {.tabset}


**Fig. 3** Infection outcomes in zebrafish exposed to Pseudocapillaria tomentosa. (A) Infection outcome analysis of fish exposed to P. tomentosa (n = 89) by temperature. Fish reared at 28°C and 32°C water temperatures had significantly different infection burden to fish reared at 35°C water temperature. Only one fish in our microbiome analysis reared at 35°C was identified as being positively infected by wet mount. Only statistically significant relationships are shown. A “*” indicates statistical significance below the “0.05” level. (B) Histological sections stained with H&E stain in zebrafish exposed to P. tomentosa examined at 35°C at 21 days post exposure. Arrow = larval worms, sagittal and cross sections. Bar = 50 µm.



#### 3A) {.active}

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP"

```

```{r plots-3A, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- worm.plots[[tmp.subset]][[tmp.variables]][["TUKEY_GLM.NB"]][["Plot"]] + 
  theme(legend.position = "none") +
  labs(title = NULL)

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_3A"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```


##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-3A}

```


###### GLM

```{r stats-3A-glm, echo=FALSE}

worm.stats[[tmp.subset]][[tmp.variables]][["GLM.NB.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### ANOVA

```{r stats-3A-anova, echo=FALSE}

worm.stats[[tmp.subset]][[tmp.variables]][["ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Tukey

```{r stats-3A-tukey, echo=FALSE}

worm.stats[[tmp.subset]][[tmp.variables]][["TUKEY_GLM.NB.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

#### 3B)

```{r plots-3B, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE }

knitr::include_graphics(file.path(proj.path,"Manuscript/Main_Figures/PNG/Fig_3B.png"))

```


##### {.tabset}

###### (hide)

###### Pathology Results

```{r}

worm.plots[[tmp.subset]][[tmp.variables]][["PathologyResults"]][["Table"]]

```


###### Total Worm Counts

```{r}

worm.plots[[tmp.subset]][[tmp.variables]][["TotalWormCount"]][["Table"]]

```


##### Suppl. {.tabset}

Click on tabs to reveal supplementary figures and tables.

### {.tabset}

#### (hide)

Click on tabs to reveal figures displaying supplementary figures and tables

#### S3A.1

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:DPE"

```

```{r plots-S3A-1, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

worm.plots[[tmp.subset]][[tmp.variables]][["TUKEY_GLM.NB"]][["Plot"]]

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-S3A-1}

```


###### GLM

```{r stats-S3A-1-glm, echo=FALSE}

worm.stats[[tmp.subset]][[tmp.variables]][["GLM.NB.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### ANOVA

```{r stats-S3A-1-anova, echo=FALSE}

worm.stats[[tmp.subset]][[tmp.variables]][["ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Tukey

```{r stats-S3A-1-tukey, echo=FALSE}

worm.stats[[tmp.subset]][[tmp.variables]][["TUKEY_GLM.NB.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

#### S3A.2

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:DPE"

```

```{r plots-S3A-2, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

worm.plots[[tmp.subset]][[tmp.variables]][["TUKEY_GLM.NB"]][["Plot_v2"]]

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-S3A-2}

```


###### GLM

```{r stats-S3A-2-glm, echo=FALSE}

worm.stats[[tmp.subset]][[tmp.variables]][["GLM.NB.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### ANOVA

```{r stats-S3A-2-anova, echo=FALSE}

worm.stats[[tmp.subset]][[tmp.variables]][["ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

#### S3C {.tabset}


```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:DPE"

```

```{r plots-S3-MethodCount-1, fig.height=6.5, fig.width=12, message=F, warning=FALSE, echo=FALSE}
worm.plots[[tmp.subset]][[tmp.variables]][["Method.Counts"]][["Plot"]]
```

##### (hide)

```{r}

```

##### Counts

```{r}

worm.stats[[tmp.subset]][["Method.Counts"]][["Detection_Counts"]]

```

##### Stats

```{r}

worm.stats[[tmp.subset]][["Method.Counts"]][["Method_Comparison_Stats"]]


```



### Fig. 4) Gut microbiome response to parasite exposure varies across water temperature {.tabset}

**Fig. 4** Effects of Pseudocapillaria tomentosa exposure on zebrafish gut microbiomes reared at different water temperatures. (A) Simpson’s Index of diversity shows that gut microbiome diversity significantly differs between fish reared at 28°C water temperature to fish reared at 32°C and 35°C water temperatures. (B) Capscale ordination based on the Bray-Curtis dissimilarity of gut microbiome composition constrained on the main effect of temperature. The analysis shows that gut microbiome composition significantly differs between parasite exposed fish reared at different water temperatures. (C) Simpson’s Index of diversity shows microbial gut diversity decreases with time from 0 days post exposure (dpe) to 42 dpe in parasite exposed fish reared at 28°C water temperature. (D) Capscale ordination of gut microbiome composition based on the Canberra dissimilarity constrained on the main effects of water temperature and time (days post exposure, dpe), and their interaction. The analysis shows that shows that gut microbiome composition differs between parasite exposed fish across time depending on water temperature. Ribbons and ellipses indicate 95% confidence interval. Only statistically significant relationships are shown. A “*” indicates statistical significance below the “0.05” level. Black arrows indicate direction of greatest change in the indicated covariates.

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.subset <- "Exposed"

```

#### 4A)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.variables <- "TEMP"
```


```{r plots-4A, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- alpha.plots[[tmp.subset]][[tmp.variables]][["TUKEY"]][["Plot"]] +
  theme(legend.position = "none") +
  labs(title = NULL)

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_4A"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-4A}

```


###### GLM

```{r stats-4A-glm, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["GLM.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### ANOVA

```{r stats-4A-anova, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Tukey

```{r stats-4A-tukey, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics

#### 4B)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.variables <- "TEMP"

```

```{r plots-4B, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_4B"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-4B}

```


###### ADONIS2

```{r stats-4B-adonis2, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (ANOVA)

```{r stats-4B-anova, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (Tukey)

```{r stats-4B-tukey, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics

#### 4C)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.variables <- "TEMP:DPE"

```

```{r plots-4C, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- alpha.plots[[tmp.subset]][[tmp.variables]][["TUKEY"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_4C"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-4C}

```


###### GLM

```{r stats-4C-glm, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["GLM.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### ANOVA

```{r stats-4C-anova, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Tukey

```{r stats-4C-tukey, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics

#### 4D)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.variables <- "TEMP:DPE"

```

```{r plots-4D, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_4D"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-4D}

```


###### ADONIS2

```{r stats-4D-adonis2, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (ANOVA)

```{r stats-4D-anova, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (Tukey)

```{r stats-4D-tukey, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```


##### Addl. Metrics

### {.tabset}

#### (hide)

Click on tabs to reveal figures displaying additional metrics.


#### S4A


```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP"

```

```{r plots-S4A, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}
alpha.plots[[tmp.subset]][[tmp.variables]][["TUKEY"]][["Plot_SUPP"]]
```

#### S4B

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP"

```

```{r plots-S4B, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP"]]


```


#### S4C


```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:DPE"

```

```{r plots-S4C, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}
alpha.plots[[tmp.subset]][[tmp.variables]][["TUKEY"]][["Plot_SUPP"]]
```

#### S4D

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:DPE"

```

```{r plots-S4D, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP"]]

```

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:DPE__TIME"

```

```{r plots-S4D.2, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP"]]

```


### Fig. 5) Gut microbiome response has a non-linear relationship with infection burden {.tabset}

**Fig. 5** The impacts of presence of infection and infection burden on the gut microbiomes of Pseudocapillaria tomentosa exposed zebrafish. (A) Simpson's Index for diversity of parasite exposed fish. Gut microbial alpha-diversity does not significantly differ between fish reared at the same water temperature depending on presence of infection. (B) Capscale ordination based on the Canberra dissimilarity of gut microbiome composition of parasite exposed fish constrained on the main effects of temperature and pathology result. The analysis shows that gut microbiome composition significantly differs between positively infected fish reared at different water temperatures. (C) Infection burden (total worm counts ) is positively correlated with lowest or highest alpha diversity scores in positively infected fish. (D) Capscale ordination based on the Bray-Curtis dissimilarity of gut microbiome composition constrained on the main effects of water temperature and infection burden. The analysis shows that gut microbiome composition significantly differs between clusters of Low, High and Other fish. Samples points are colored by water temperature, and filled by cluster grouping. Samples with at least one detectable worm and an alpha-diversity score less than 0.5 are categorized as Low (orange fill), samples with at least one detectable worm and an alpha-diversity score greater than 0.5 are categorized as High (purple fill), and samples with no observable infection are categorized as Other (white and transparent fill). Ribbons and ellipses indicate 95% confidence interval. Only statistically significant relationships are shown. A "*" indicates statistical significance below the "0.05" level. Black arrows indicate statistically significant covariates and direction of greatest change in the indicated covariates.

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.subset <- "Exposed"
tmp.variables <- "TEMP:PATH"

```

#### 5A)

```{r plots-5A, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- alpha.plots[[tmp.subset]][["TEMP:PATH"]][["TUKEY"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_5A"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-5A}

```


###### GLM

```{r stats-5A-glm, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["GLM.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### ANOVA

```{r stats-5A-anova, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Tukey

```{r stats-5A-tukey, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```


##### Addl. Metrics

#### 5B)

```{r plots-5B, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- beta.plots[[tmp.subset]][["TEMP:PATH"]][["CAP"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_5B"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-5B}

```


###### ADONIS2

```{r stats-5B-adonis2, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (ANOVA)

```{r stats-5B-anova, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (Tukey)

```{r stats-5B-tukey, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics

#### 5C)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:CLUSTER"

```

```{r plots-5C, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- alpha.plots[[tmp.subset]][["TEMP:CLUSTER"]][["Cluster"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_5C"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-5C}

```


###### GLM

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:CLUSTER"

```

```{r stats-5C1-glm, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["GLM.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:WORM"

```

```{r stats-5C2-glm, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["GLM.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### ANOVA

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:CLUSTER"

```

```{r stats-5C1-anova, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:WORM"

```

```{r stats-5C2-anova, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Tukey

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:CLUSTER"

```

```{r stats-5C1-tukey, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```



##### Addl. Metrics

#### 5D)


```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:CLUSTER"

```

```{r plots-5D, fig.height=6.5, fig.width=6.5, message=FALSE, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_5D"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-5D}

```


###### ADONIS2

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:CLUSTER"

```

```{r stats-5D1-adonis2, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:WORM"

```

```{r stats-5D2-adonis2, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (ANOVA)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:CLUSTER"

```

```{r stats-5D-anova, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (Tukey)

```{r stats-5D-tukey, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics

### {.tabset}

#### (hide)

Click on tabs to reveal figures displaying additional metrics.


#### S5A


```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:PATH"

```

```{r plots-S5A, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}
alpha.plots[[tmp.subset]][[tmp.variables]][["TUKEY"]][["Plot_SUPP"]]
```


#### S5B


```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:PATH"

```

```{r plots-S5B, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP"]]

```

#### S5C

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:CLUSTER"

```

```{r plots-S5C, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

alpha.plots[[tmp.subset]][[tmp.variables]][["TUKEY"]][["Plot_SUPP"]]

```

#### S5D {.tabset}

Click on tabs to cycle through different cluster comparisons between alpha and beta diversity.

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
tmp.variables <- "TEMP:CLUSTER"

```

##### Phylogenetic

```{r plots-S5D-phylogenetic, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP_phylogenetic"]]

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS__SUPP_5D.1"]] %>%
  filter(Alpha.Metric == "Phylogenetic") %>%
    set_GT(var = "p.value", group.by = "Alpha.Metric") %>%

    # Title/caption
    gt::tab_header(
      title = "ADONIS2",
      subtitle = "adonis2(Beta Distance ~ Cluster); Exposed fish"
    )

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey__SUPP_5D.1"]] %>%
  filter(Alpha.Metric == "Phylogenetic") %>%
    set_GT(var = "p.value", group.by = "Alpha.Metric") %>%

      # Title/caption
      gt::tab_header(
        title = "ANOVA: Homogeneity of Dispersion",
        subtitle = "ANOVA(Beta Dispersion ~ Cluster); Exposed fish"
        )

```


##### Richness

```{r plots-S5D-richness, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP_richness"]]

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS__SUPP_5D.1"]] %>%
  filter(Alpha.Metric == "Richness") %>%
    set_GT(var = "p.value", group.by = "Alpha.Metric") %>%

    # Title/caption
    gt::tab_header(
      title = "ADONIS2",
      subtitle = "adonis2(Beta Distance ~ Cluster); Exposed fish"
    )

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey__SUPP_5D.1"]] %>%
  filter(Alpha.Metric == "Richness") %>%
    set_GT(var = "p.value", group.by = "Alpha.Metric") %>%

      # Title/caption
      gt::tab_header(
        title = "ANOVA: Homogeneity of Dispersion",
        subtitle = "ANOVA(Beta Dispersion ~ Cluster); Exposed fish"
        )

```

##### Shannon

```{r plots-S5D-shannon, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP_shannon"]]

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS__SUPP_5D.1"]] %>%
  filter(Alpha.Metric == "Shannon") %>%
    set_GT(var = "p.value", group.by = "Alpha.Metric") %>%

    # Title/caption
    gt::tab_header(
      title = "ADONIS2",
      subtitle = "adonis2(Beta Distance ~ Cluster); Exposed fish"
    )

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey__SUPP_5D.1"]] %>%
  filter(Alpha.Metric == "Shannon") %>%
    set_GT(var = "p.value", group.by = "Alpha.Metric") %>%

      # Title/caption
      gt::tab_header(
        title = "ANOVA: Homogeneity of Dispersion",
        subtitle = "ANOVA(Beta Dispersion ~ Cluster); Exposed fish"
        )

```

##### Simpsons {.active}

```{r plots-S5D-simpsons, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP_simpsons"]]

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS__SUPP_5D.1"]] %>%
  filter(Alpha.Metric == "Simpson") %>%
    set_GT(var = "p.value", group.by = "Alpha.Metric") %>%

    # Title/caption
    gt::tab_header(
      title = "ADONIS2",
      subtitle = "adonis2(Beta Distance ~ Cluster); Exposed fish"
    )

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey__SUPP_5D.1"]] %>%
  filter(Alpha.Metric == "Simpson") %>%
    set_GT(var = "p.value", group.by = "Alpha.Metric") %>%

      # Title/caption
      gt::tab_header(
        title = "ANOVA: Homogeneity of Dispersion",
        subtitle = "ANOVA(Beta Dispersion ~ Cluster); Exposed fish"
        )

```


### {.tabset}

**Cluster Counts**

Counts of samples per cluster and temperature by alpha-diversity metric. Click on tabs to reveal tables.

#### (hide)

#### Simpson

```{r stats-5S-counts_simpson, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["Counts_Temp.Cluster"]] %>%
  dplyr::filter(Alpha.Metric == "Simpson") %>%
  gt::gt()

```

#### Shannon

```{r stats-5S-counts_shannon, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["Counts_Temp.Cluster"]] %>%
  dplyr::filter(Alpha.Metric == "Shannon") %>%
  gt::gt()

```

#### Richness

```{r stats-5S-counts_richness, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["Counts_Temp.Cluster"]] %>%
  dplyr::filter(Alpha.Metric == "Richness") %>%
  gt::gt()

```

#### Phylogenetic

```{r stats-5S-counts_phylogenetic, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["Counts_Temp.Cluster"]] %>%
  dplyr::filter(Alpha.Metric == "Phylogenetic") %>%
  gt::gt()

```


### Fig. 6) Parasite exposure exacerbates water temperature differences in gut microbiome structure {.tabset}


**Fig. 6** Comparison of the effects of water temperature on the gut microbiome between parasite exposed fish and parasite unexposed fish. (A) Simpson's Index for diversity of parasite unexposed and pre-exposed fish at 0 days post exposure (dpe). Prior to parasite exposure gut microbial alpha-diversity does not significantly differ between fish reared at the same water temperature. (B) Capscale ordinations based on the Bray-Curtis dissimilarity of gut microbiome composition constrained on the main and interaction effects of temperature and parasite exposure (treatment) of pre-exposure samples at 0 dpe. (C) Simpson's Index for diversity of parasite unexposed and exposed fish. Gut microbial alpha-diversity significantly differs between parasite exposed fish reared at 28°C and 32°C water temperature relative to unexposed control fish, but gut microbial alpha-diversity does not differ between parasite unexposed and exposed fish reared at 35°C water temperature. (D) Capscale ordinations based on the Bray-Curtis dissimilarity of gut microbiome composition constrained on the main and interaction effects of temperature and parasite exposure (treatment) of post-exposure samples after 0 dpe. The analysis shows gut microbiome composition differs between fish reared at different water temperatures prior to parasite exposure, and parasite exposure further drives these temperature associated differences in microbiome community composition. Ribbons and ellipses indicate 95% confidence interval. Ribbons and ellipses indicate 95% confidence interval. Only statistically significant relationships are shown. A "*" indicates statistical significance below the "0.05" level. Black arrows indicate statistically significant covariates and direction of greatest change in the indicated covariates.


#### 6A)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.subset <- "PreExposed"
tmp.variables <- "TEMP:TREAT"

```

```{r plots-6A, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- alpha.plots[["PreExposed"]][["TEMP:TREAT"]][["TUKEY"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_6A"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-6A}

```


###### GLM

```{r stats-6A-glm, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["GLM.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### ANOVA

```{r stats-6A-anova, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Tukey

```{r stats-6A-tukey, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics


#### 6B)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.subset <- "PreExposed"
tmp.variables <- "TEMP:TREAT"

```

```{r plots-6B, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- beta.plots[["PreExposed"]][["TEMP:TREAT"]][["CAP"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_6B"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-6B}

```

###### ADONIS2

```{r stats-6B-adonis2, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (ANOVA)

```{r stats-6B-anova, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (Tukey)

```{r stats-6B-tukey, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics

#### 6C)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.subset <- "PostExposed"
tmp.variables <- "TEMP:TREAT"

```

```{r plots-6C, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- alpha.plots[["PostExposed"]][["TEMP:TREAT"]][["TUKEY"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_6C"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-6C}

```


###### GLM

```{r stats-6C-glm, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["GLM.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### ANOVA

```{r stats-6C-anova, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Tukey

```{r stats-6C-tukey, echo=FALSE}

alpha.stats[[tmp.subset]][[tmp.variables]][["Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics


#### 6D)

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.subset <- "PostExposed"
tmp.variables <- "TEMP:TREAT"

```

```{r plots-6D, fig.height=6.5, fig.width=6.5, message=F, warning=FALSE, echo=FALSE}

# Store the plot in a temporary variable for saving
p <- beta.plots[["PostExposed"]][["TEMP:TREAT"]][["CAP"]][["Plot"]]

# Display the plot
p



# Extract figure number from chunk name
fig_name <- "Fig_6D"

# Save as PNG and PDF
ggsave(filename = file.path(png_dir, paste0(fig_name, ".png")), 
       plot = p, width = 6.5, height = 6.5, dpi = 300)
ggsave(filename = file.path(pdf_dir, paste0(fig_name, ".pdf")), 
       plot = p, width = 6.5, height = 6.5)

```

##### Tables {.tabset}

###### (hide)

Click on tabs to display tables. Scroll to see additional rows.

```{r empty-6D}

```

###### ADONIS2

```{r stats-6D-adonis2, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (ANOVA)

```{r stats-6D-anova, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.ANOVA.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (Tukey)

```{r stats-6D-tukey, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey.Table"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

##### Addl. Metrics

### {.tabset}

#### (hide)

Click on tabs to reveal figures displaying additional metrics.


#### S6A


```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "PreExposed"
tmp.variables <- "TEMP:TREAT"

```

```{r plots-S6A, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}
alpha.plots[[tmp.subset]][[tmp.variables]][["TUKEY"]][["Plot_SUPP"]]
```

#### S6B

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "PreExposed"
tmp.variables <- "TEMP:TREAT"

```

```{r plots-S6B, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP"]]

```

##### Suppl. Plots {.tabset}

###### (hide)

Click on tabs to display tables and figures. Scroll to see additional rows.

###### By Temp


```{r plots-6B.1_byTemp, echo=FALSE, fig.height=4, fig.width=6.5, message=FALSE, warning=FALSE}

# Note to future self (from 03-21-25): If the code breaks here it's because I renamed 6B and 6C to be congruent with the order they appear in the manuscript. For the plots to appear correctly in the supplemental figures, the code generating the stats and plots for section 6B and 6C need to be reran and the Robjects saved prior to running this markdown file. I've already correctly renamed the variables (I think) here. 

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP_6B.1_bray"]]

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP_6B.1_canberra"]]

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP_6B.1_gunifrac"]]

```

##### Suppl. Tables (by Temp) {.tabset}

###### (hide)

Click on tabs to display tables and figures. Scroll to see additional rows.

```{r empty-6B.1_hide}

```

###### ADONIS2

```{r stats-6B.1-adonis2, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS__SUPP_6B.1"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (ANOVA)

```{r stats-6B.1-anova, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey__SUPP_6B.1"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```



##### Suppl. Tables (by Tank) {.tabset}

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "PreExposed"
tmp.variables <- "TEMP:TANK"

```

###### (hide)

Click on tabs to display tables and figures. Scroll to see additional rows.

```{r empty-6B.1.1_hide}

```

###### ADONIS2

```{r stats-6B.1.1-adonis2, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS__SUPP_6B.2.2"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (ANOVA)

```{r stats-6B.1.1-anova, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey__SUPP_6B.3.2"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```


#### S6C


```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "PostExposed"
tmp.variables <- "TEMP:TREAT"

```

```{r plots-S6C, fig.height=6.5, fig.width=14, message=F, warning=FALSE, echo=FALSE}
alpha.plots[[tmp.subset]][[tmp.variables]][["TUKEY"]][["Plot_SUPP"]]
```

#### S6D

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "PostExposed"
tmp.variables <- "TEMP:TREAT"

```

```{r plots-S6D, echo=FALSE, fig.height=6.5, fig.width=14, message=FALSE, warning=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP"]]

```

##### Suppl. Plots {.tabset}

###### (hide)

Click on tabs to display tables and figures. Scroll to see additional rows.


###### By Temp

```{r plots-6D.1_byTemp, echo=FALSE, fig.height=4, fig.width=6.5, message=FALSE, warning=FALSE}

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP_6D.1_bray"]]

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP_6D.1_canberra"]]

beta.plots[[tmp.subset]][[tmp.variables]][["CAP"]][["Plot_SUPP_6D.1_gunifrac"]]

```

##### Suppl. Tables {.tabset}

###### (hide)

Click on tabs to display tables and figures. Scroll to see additional rows.

```{r empty-6D.1_hide}

```

###### ADONIS2

```{r stats-6D.1-adonis2, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["CAP.ADONIS__SUPP_6D.1"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```

###### Dispersion (ANOVA)

```{r stats-6D.1-anova, echo=FALSE}

beta.stats[[tmp.subset]][[tmp.variables]][["HoD.Tukey__SUPP_6D.1"]] %>%
  # Set the maximum height to 6.5 inches (6.5 * 96 pixels = 624 pixels)
    gt::tab_options(
      container.overflow.y = "auto",       # Enable vertical overflow
      container.height = gt::px(624)           # Set the height to 624 pixels (6.5 inches)
    )

```



### Fig. 7) Gut microbial abundance is significantly associated with environmental conditions and stressors

```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document

tmp.subset <- "All"

```

#### 7) Gut microbial abundance is significantly associated with environmental conditions and stressors

**Fig. 7** A heatmap of model coefficient values of the top 50 statistically significant abundant gut microbial taxa identified by MaAsLin2. The color of each cell represents the coefficient value and direction (red is positive, blue is negative). A "+" or "-" indicates a statistically significant association was observed between taxon abundance and a covariate. Gray colored cells indicate a significant effect was not observed.

```{r plots-7, out.width="90%", echo=FALSE}

# Store the plot in a temporary variable for saving
gt_table <- diffAbnd.plots[["All"]][["All__TEMP_DPE_TREAT_PATH_WORM_CLUSTER"]][["Maaslin2"]][["Top50"]]

# Display the plot 
gt_table



# Extract figure number from chunk name
fig_name <- "Fig_7"


# Save as PNG and PDF
# gt::gtsave(gt_table, filename = file.path(png_dir, paste0(fig_name, ".png")))
# gt::gtsave(gt_table, filename = file.path(pdf_dir, paste0(fig_name, ".pdf")))


```


##### Tables {.tabset}

###### (hide)

Click on tabs to reveal tables displaying additional information.

###### S7A.1
```{r tables-S7X-1}
# Store the gt table
gt_table <- diffAbnd.plots[["All"]][["All__TEMP_DPE_TREAT_PATH_WORM_CLUSTER"]][["Maaslin2"]][["SUPP__SigCounts"]]

# Display the table
gt_table

# Save as PNG and PDF
# gt::gtsave(gt_table, filename = file.path(png_dir, "S7-1.png"))
# gt::gtsave(gt_table, filename = file.path(pdf_dir, "S7-1.pdf"))
```

###### S7A.2
```{r tables-S7X-2}
# Store the gt table
gt_table <- diffAbnd.plots[["All"]][["All__TEMP_DPE_TREAT_PATH_WORM_CLUSTER"]][["Maaslin2"]][["SUPP__SigTaxonomy"]]

# Display the table
gt_table

# Save as PNG and PDF
# gt::gtsave(gt_table, filename = file.path(png_dir, "S7-2.png"))
# gt::gtsave(gt_table, filename = file.path(pdf_dir, "S7-2.pdf"))
```

##### Suppl. {.tabset}

###### (hide)

Click on tabs to reveal figures displaying additional metrics.

###### S7A.1
```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "All"
```

```{r plots-S7X, echo=FALSE}
# Store the gt table
gt_table <- diffAbnd.plots[["All"]][["All__TEMP_DPE_TREAT_PATH_WORM_CLUSTER"]][["Maaslin2"]][["All_sig"]] %>%
  gt::tab_options(
    container.overflow.y = "auto",
    container.height = gt::px(1024)
  )

# Display the table
gt_table

# Save as PNG and PDF
# gt::gtsave(gt_table, filename = file.path(png_dir, "S7X.png"))
# gt::gtsave(gt_table, filename = file.path(pdf_dir, "S7X.pdf"))
```
###### S7B.1
```{r include=FALSE}
# creates a temporary variable to aid in automating knitted document
tmp.subset <- "Exposed"
```

```{r plots-S7B-1, echo=FALSE}

randomForest.plots[[tmp.subset]][["RandomForest"]][["Top25_Features"]]

# Save as PNG and PDF
# gt::gtsave(gt_table, filename = file.path(png_dir, "S7X.png"))
# gt::gtsave(gt_table, filename = file.path(pdf_dir, "S7X.pdf"))
```

```{r stats-S7B-1, echo=FALSE}

randomForest.stats[["Exposed"]][["RandomForest"]][["Performance"]]
randomForest.stats[["Exposed"]][["RandomForest"]][["Top25_Features"]]
randomForest.stats[["Exposed"]][["RandomForest"]][["StabilityAnalysis_Top10"]]
randomForest.stats[["Exposed"]][["RandomForest"]][["CrossValidation_Summary_10Fold"]]
randomForest.stats[["Exposed"]][["RandomForest"]][["CrossValidation_Improvement"]]

```

## Session Info {.tabset}

### {.tabset}

Click through tabs to reveal session information.

#### (hide)

#### Session Info

```{r session-info, echo=FALSE}
# Display session info
session_info
```




